src/ontology/foodon-edit.owl

```diff
@@ -31444,6 +31563,18 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_03306973 -->
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FOODON_03306973">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza with Pepperoni, which is an American variety of salami, made from cured pork and beef seasoned with paprika or other chili pepper.</obo:IAO_0000115>
```

Comment:
Pepperoni shouldn't be defined here, it should have its own entry.
This would be defined as "A pizza with pepperoni topping"